packages/cli-wrapper/jest.config.js

```diff
@@ -1,6 +1,6 @@
-//TODO: Somehow move this to the parent folder and have children use this config
+// TODO: Somehow move this to the parent folder and have children use this config
 /** @type {import('@ts-jest/dist/types').InitialOptionsTsJest} */
 module.exports = {
-  preset: 'ts-jest',
-  testEnvironment: 'node',
-};
\ No newline at end of file
+  preset: "ts-jest",
```

Comment:
By the way you have any idea of how we could possibly share jest configs? Because right now there is a jest config in each file which makes the project quite unmanageable... I know rush.js can but that migration is alot of work also

Reply:
We can import jest configs from parent folders like this https://stackoverflow.com/questions/40726702/is-there-anyway-to-extend-a-jest-configuration-file

Reply:
Alright let me submit a PR